delocate/delocating.py

```diff
@@ -499,7 +556,7 @@ def delocate_wheel(
         to sequence ``['x86_64, 'i386']``) or name of required architecture
         (e.g "i386" or "x86_64").
     check_verbose : bool, optional
-        If True, print warning messages about missing required architectures
+        This flag is currently ignored.
```

Comment:
I guess we should decide on deprecation here.  Maybe we could set this to None by default, and raise a deprecation warning if it is not None.    Maybe warn too that this and subsequent arguments will become keyword-only in the future.

Reply:
It can be hard to switch a function over to keyword only arguments.  Only `FutureWarning` is loud enough that you can expect people using the older API to notice that it's actually deprecated.

Reply:
I've marked it with a DeprecationWarning and labeled the remaining parameters as keyword-only.

Reply:
Thanks.  I'll follow up with a little PR to check the further parameters.